validator/client/beacon-api/propose_beacon_block.go

```
@@ -89,15 +120,23 @@ func handlePhase0Block(block *ethpb.GenericSignedBeaconBlock_Phase0) (*blockProc
 	}
 	res.beaconBlockRoot = beaconBlockRoot
 
-	signedBlock := structs.SignedBeaconBlockPhase0FromConsensus(block.Phase0)
-	res.marshalledJSON, err = json.Marshal(signedBlock)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to marshall phase0 beacon block to json")
+	// Marshal SSZ
+	res.marshalledSSZ, _ = block.Phase0.MarshalSSZ()
```

Any reason for ignoring the error?

---

that shouldn't be the case oops